dbfit-java/core/src/main/java/dbfit/fixture/FunctionResultSetStatementExecution.java

```diff
@@ -1,14 +1,12 @@
-package dbfit.environment;
+package dbfit.fixture;
```

Comment:
That's the same as the base class. In that other big API related PR I think we moved it (StatementExecution`) out to somewhere else.